core/iwasm/aot/aot_runtime.c

```diff
@@ -5020,3 +5020,121 @@ aot_get_module_name(AOTModule *module)
 {
     return module->name;
 }
+
+void
+aot_resolve_symbols(AOTModule *module)
+{
+    for (uint32 idx = 0; idx < module->import_func_count; ++idx) {
```

Comment:
had better declare `uint32 idx` first, it is a C source file, we usually do that way